### PR TITLE
fix hourly bell sound in undead zone

### PR DESCRIPTION
**Pull request by schell244**
fixes issue:
https://github.com/vmangos/core/issues/344

### CODE REVIEW

**Comment by schell244** [comment]
Btw: there is no bell gameobject in undercity, so the sound will only be played in Brill
(but if there would be added a bell in future, it should trigger the correct sound).

File: src/scripts/world/go_scripts.cpp [hunk 1]
```diff
@@ -372,7 +372,8 @@ enum BellHourlySoundFX
 
 enum BellHourlySoundAreas
 {
-    UNDERCITY_AREA = 1497,
+    TIRISFAL_ZONE = 85,
+	UNDERCITY_ZONE = 1497,
```
Comment (Kittnz) [inline src/scripts/world/go_scripts.cpp]: tabs

**Comment by ratkosrb** [comment]
Change tabs to 4 spaces in visual studio settings.
https://camo.githubusercontent.com/d1f8fdb8d2efba08ec0dc333881b91e1b79a354a/68747470733a2f2f692e696d6775722e636f6d2f4c593672486e4e2e6a7067

**Comment by schell244** [comment]
Thank you. I'm not experienced with c++ & visial studio at all. I will update the other tabs as well then.

**Comment by schell244** [comment]
whops, the other tabs are already 4 spaces.

**Comment by schell244** [comment]
I went to all spawn points in db -  now all hourly played sounds are correct.
TODO:
 add bell gameobject spawn in Undercity (id:175885) and Darnassus (id:176573) to DB

**Comment by Kittnz** [comment]
yeah i think we are missing a few spawns around the world

**Comment by schell244** [comment]
We currently have spawns:
brill, orgrimmar, thunderbluff, scarlet monastery, tarren mill, darkshire,
stormwind, ironforge, northshire abbey, westfal lighthous, astranaar

**Comment by schell244** [comment]
...by the way for testing you can just trigger the event: 
 .event start 78
 .event stop 78
I just figured this out, before I changed my system time :D

**Comment by Kittnz** [comment]
ofcourse ^^

**Comment by ratkosrb** [comment]
Is this ready to be merged?

**Comment by schell244** [comment]
From my side, it's ready

**Pull request merged**